Ory/kratos/kratos.yml

```diff
@@ -23,47 +19,32 @@ serve:
       exposed_headers:
         - Content-Type
         - Set-Cookie
-  admin:
-    base_url: http://kratos.openfoodfacts.localhost:4434/
 
 selfservice:
-  default_browser_return_url: http://world.openfoodfacts.localhost/
-  allowed_return_urls:
-    - http://world.openfoodfacts.localhost/
+  default_browser_return_url: http://world.openfoodfacts.localhost/cgi/kratos_auth.pl
```

Comment:
Is this the last "localhost" hardcoded url ? Is it harder to configure through environment ?

Reply:
This does not matter as it is just the default, as long as they are configured through env in docker file all is good.